### ISSUE TITLE
Download

### ISSUE THREAD

**Issue by lfilor** [szabgab/App-PerlTidy-Tk#1]
Hi Gabor,

tried your command to download - but got an error message.
Could be a problem on my side, or something hidden

![Capture](https://user-images.githubusercontent.com/3590366/93002852-bb4cfc80-f4ee-11ea-89f3-351e82c2c24a.PNG)
![Capture](https://user-images.githubusercontent.com/3590366/93002874-e33c6000-f4ee-11ea-80ee-8635b196613e.PNG)

**Comment by szabgab** [comment]
It seem it takes several hours till the cpanmetadb gets updates. I also had the same problem a few hours ago (even when I was recording the video, that's why I did not show this part), but I just checked and it already works for me. (About 2 hours after you reported this.) So try it again and let me know.

**Comment by lfilor** [comment]
Hi Gabor, - I waited till the next day, but I still get the same error.Try to follow your tutorial, and help to pipe clean. If there is anythingelse to tryout let me know. Until now I am stuck and can't replicate.
- Lutz
    On Saturday, September 12, 2020, 01:45:21 PM PDT, Gabor Szabo <notifications@github.com> wrote:  
 
 


It seem it takes several hours till the cpanmetadb gets updates. I also had the same problem a few hours ago (even when I was recording the video, that's why I did not show this part), but I just checked and it already works for me. (About 2 hours after you reported this.) So try it again and let me know.

**Comment by szabgab** [comment]
Oh, I just noticed in the command you typed in a : is missing.
```
cpanm App::PerlTidy::Tk
```

and you'll probably also need to run

```
cpanm Tk::HyperText
```

before, as I forgot yo add that dependency.

**Comment by lfilor** [comment]
Oh boy - you are right, shouldn't work on two projects at the same time,when you said it - it worked right away
But with 
still getting
Retry with --force


    On Sunday, September 13, 2020, 09:22:41 AM PDT, Gabor Szabo <notifications@github.com> wrote:  
 
 


Oh, I just noticed in the command you typed in a : is missing.
cpanm App::PerlTidy::Tk

and you'll probably also need to run
cpanm Tk::HyperText

before, as I forgot yo add that dependency.

**Comment by lfilor** [comment]
The devil is in the detail. Everybody has a different setup and is missing somemodules here is what I just figured out


Hope that this will help your course to be more successful and helps other nubiesto see the fantastic work you do.
Cheers- Lutz
    On Sunday, September 13, 2020, 09:22:41 AM PDT, Gabor Szabo <notifications@github.com> wrote:  
 
 


Oh, I just noticed in the command you typed in a : is missing.
cpanm App::PerlTidy::Tk

and you'll probably also need to run
cpanm Tk::HyperText

before, as I forgot yo add that dependency.
